Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -3987,6 +3991,21 @@ void CClientPed::_ChangeModel()
             _CreateModel();
         }
 
+        // ReAttach satchels
+        static CClientProjectileManager* pProjectileManager = m_pManager->GetProjectileManager();
+
+        for (const SSatchelsData& satchelData : attachedSatchels)
+        {
+            CClientProjectile* pSatchel = pProjectileManager->Get((CEntitySAInterface*)satchelData.pProjectileInterface);
+            if (!pSatchel || pSatchel->IsBeingDeleted())
+                continue;
+
+            pSatchel->SetAttachedOffsets(*satchelData.vecAttachedOffsets, *satchelData.vecAttachedRotation);
+            pSatchel->InternalAttachTo(this);
+        }
+
+        attachedSatchels.clear();
```

Comment:
This can be removed since `attachedSatchels` won't be used down to the function's end.